capstone/src/main/webapp/youtubeScript.js

```diff
@@ -34,8 +34,13 @@ const YOUTUBE_CATEGORIES = {
   'Sports' : 17,
 };
 
+/** These variables will keep track of the data required for CSV output */
+let attributeData = [];
```

Comment:
nit: `attributeData` -> `ATTRIBUTE_DATA` and `analyzedComments` -> `ANALYZED_COMMENTS`

Reply:
addressed in "address Daniel's and Chris's nits"